pennylane/ops/qubit/time_dependent_hamiltonian.py

```diff
@@ -0,0 +1,218 @@
+"""
+This submodule contains the time-dependent hamiltonian class
+"""
+
+from jax import numpy as jnp
+
+import pennylane as qml
+from pennylane.operation import Observable, Tensor
+from pennylane.ops.qubit.hamiltonian import Hamiltonian
+
+
+# pylint: disable= too-many-instance-attributes
+class TDHamiltonian(Observable):
```

Comment:
Sorry for the confusion, this was my bad, but it is supposed to be more general `ParametrizedHamiltonian`

Reply:
No worries, easy to fix :)